docker/install/ubuntu_install_arduino.sh

```diff
@@ -23,7 +23,7 @@ set -o pipefail
 export DEBIAN_FRONTEND=noninteractive
 apt-get install -y ca-certificates
 
-ARDUINO_CLI_VERSION="0.18.3"
+ARDUINO_CLI_VERSION="0.21.1"
```

Comment:
can you consolidate arduino CLI installation into one script and reuse for both docker and RVM?

Reply:
We decided last summer that the scripts should be separate, as they aren't that similar. I'd be open to revisiting that topic, but it's OOS for this PR.